rln/src/utils.rs

```diff
@@ -1,34 +1,63 @@
 use ark_bn254::{Bn254, Fr, Parameters};
-use ark_ff::{Fp256, PrimeField};
+use ark_ff::{BigInteger, Field as ArkField, FpParameters, PrimeField};
 use ark_std::str::FromStr;
 use ethers_core::utils::keccak256;
 use num_bigint::{BigInt, BigUint, ToBigInt};
 use semaphore::{identity::Identity, Field};
+use std::iter::Extend;
 
-pub fn to_fr(el: Field) -> Fr {
-    Fr::try_from(el).unwrap()
+pub fn modulus_bit_size() -> usize {
+    <Fr as PrimeField>::Params::MODULUS
+        .num_bits()
+        .try_into()
+        .unwrap()
+}
+
+pub fn fr_byte_size() -> usize {
```

Comment:
Style comment. The usage of `fr`, `el` and other abbreviations feels very Fortran-esk and might not make the code as readable for the next developer or future you.

Reply:
Thanks! FYI here fr indicates Fr, i.e. the finite field for scalars, so actually is not an abbreviation!